apps/web-client/src/routes/inventory/warehouses/[...id]/+page.svelte

```diff
@@ -57,6 +57,30 @@
 
 	$: displayName = warehouesStores.displayName;
 	$: entries = warehouesStores.entries;
+	$: csvEntries = warehouesStores.csvEntries;
+
+	// #region csv
+	type CsvEntries = {
+		warehouseDiscount: number;
+		warehouseName: string;
+		isbn: string;
+		title: string;
+		price: number;
+		year?: string;
+		authors?: string;
+		publisher?: string;
+		editedBy?: string;
+		outOfPrint?: boolean;
+	};
+
+	const handleExportCsv = () => {
+		const csvConfig = mkConfig({ useKeysAsHeaders: true });
```

Comment:
It would be nice to also sepcify a `filename` here with warehouse name and timestamp.